src/main/target/FISHDRONEF4/config.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "io/vtx.h"
+
+#ifdef USE_TARGET_CONFIG
+#if defined(STACKX)
+#define VTX_SETTINGS_ULTRA_LOW_POWER          0xff
```

Comment:
This should be defined in `src/main/drivers/vtx_common.h`, and used in `src/main/drivers/vtx_rtc6705_soft_spi.c` as well.